src/metabase/search/appdb/scoring.clj

```diff
@@ -51,8 +53,10 @@
       [:- ceiling
        [:/
         ;; Use seconds for granularity in the fraction.
-        ;; TODO will probably need to specialize this based on (mdb/db-type)
-        [[:raw "EXTRACT(epoch FROM (" [:- to-column from-column] [:raw "))"]]]
+        (case (mdb/db-type)
```

Comment:
Again I'd prefer if we moved this divergence into the specialization code. Having a mix of case statements and polymorphism is a bit of a mess.